pennylane/_device.py

```diff
@@ -571,6 +573,56 @@ def my_expansion_function(self, tape, max_expansion=10):
         """
         self.custom_expand_fn = types.MethodType(fn, self)
 
+    def _local_tape_expand(self, tape, depth, stop_at):
+        """Function to recursively expand the operations in a circuit"""
```

Comment:
Even if this will just be a private helper function, we should still flesh out the documentation a little bit more.  I can imagine someone back to this code in several months and not being able to figure why this exists and what it does.

And adding some sort of comment like:
```
This function mimics `pennylane.tape.tape.expand`, but omits validation and errors for non-commuting measurements on the same wires.
```

Reply:
Added 👍🏼